invenio_administration/views/base.py

```diff
@@ -263,8 +263,7 @@ def get_context(self, pid_value=None):
         serialized_schema = self._schema_to_json(schema)
         fields = self.item_field_list
         return {
-            "request_headers"
-            "name": name,
+            "request_headers" "name": name,
```

Comment:
This looks 👀 :)

Reply:
seems like it's Missing a `,` between "request_headers", "name": name